src/data/maps/5/3/index.js

```diff
@@ -59,7 +59,7 @@ const stairs = {
 };
 
 const message = {
-    title: '<> was faced with a choice.  A short cut lied to their right.',
+    title: '<> was faced with a choice.  A shortcut lied to their right.',
```

Comment:
Sorry for not pointing this out before, but I think it really should be `A shortcut lay to their right.`